go.mod

```diff
@@ -104,3 +104,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => ./types
```

Comment:
Remove later when approved and a types PR is merged.

Reply:
Yep, go for it. Feel fee to merge the types PR and this one after answers/suggestions are taken in consideration.